db/migrate/20220824172503_create_timeline_events.rb

```diff
@@ -0,0 +1,13 @@
+class CreateTimelineEvents < ActiveRecord::Migration[7.0]
+  def change
+    create_table :timeline_events do |t|
+      t.string :event_type, null: false
+      t.references :application_form, foreign_key: true
+      t.string :title, null: false
+      t.string :annotation, null: true
```

Comment:
```suggestion
      t.string :annotation, null: false, default: ""
```

I prefer this for strings as it means there aren't two ways of representing a blank value (`nil` and `""`). Instead the blank value can only be an empty string.